server/api/batch_inventory.py

```diff
@@ -442,19 +443,81 @@ def process_batch_inventory_tabulator_status_file(
     jurisdiction = Jurisdiction.query.get(jurisdiction_id)
     batch_inventory_data = BatchInventoryData.query.get(jurisdiction_id)
 
+    def get_tabulator_id_to_name_dict_for_excel_file(
+        cvr_xml: ElementTree.ElementTree,
+    ) -> Dict[Optional[str], Optional[str]]:
+        namespaces = {"ss": "urn:schemas-microsoft-com:office:spreadsheet"}
+        worksheet = cvr_xml.find(
+            ".//ss:Worksheet[@ss:Name='Tabulator Status']", namespaces
+        )
+        if worksheet is None:
+            raise UserError(TABULATOR_STATUS_PARSE_ERROR)
+
+        tabulator_status_table = worksheet.find("ss:Table", namespaces)
+        if tabulator_status_table is None:
+            raise UserError(TABULATOR_STATUS_PARSE_ERROR)
+
+        columns = tabulator_status_table.findall("Column")
+        if columns is None:
+            raise UserError(TABULATOR_STATUS_PARSE_ERROR)
+
+        rows = tabulator_status_table.findall("ss:Row", namespaces)
+        if rows is None:
+            raise UserError(TABULATOR_STATUS_PARSE_ERROR)
+
+        # Ignore rows containing data we don't need:
+        # 1. "Tabulator Status"
+        # 2. Election name
+        # 3. "Official"/"Unofficial" status
+        # 4. Timestamp
+        # 5. Empty row for spacing
+        row_offset = 5
+        rows = rows[row_offset:]
```

Comment:
I wonder if it would be a bit more reliable to search for the headers row rather than assume its in row 6.

Reply:
https://github.com/votingworks/arlo/pull/2018/commits/abddab989b1569c29fd6d0404c4905b926c11fea#diff-6c612f15bc7de123abfc44b5aead6ce7e0c3a7765f3a0b08543553d068f0e233R464-R473 searches for the headers instead of assuming row 6